Admin/AbstractAdmin.php

```diff
@@ -766,16 +766,28 @@ public function getFilterParameters()
     {
         $parameters = array();
 
+        // if filter persister was configured with `true`, use the default persister (session)
+        if ($this->filterPersister === true) {
+            $this->filterPersister = new SessionFilterPersister($this->request->getSession());
```

Comment:
Getting rid of the boolean mode is a good thing, b/c then it will be easier to reason on the type of `filterPersister`, and use native checks like type hinting.